modules/apps/wysiwyg/wysiwyg-web/bnd.bnd

```diff
@@ -0,0 +1,6 @@
+Bundle-Name: Liferay Wysiwyg Web Conversion
+Bundle-SymbolicName: com.liferay.wysiwyg.web
+Bundle-Version: 1.0.0
+Liferay-Releng-Module-Group-Description:
+Liferay-Releng-Module-Group-Title:
+Web-ContextPath: /wysiwyg-web
```

Comment:
@JorgeFerrer I will work with Shuyang and also have him review this. I think this module should belong under journal, maybe "journal-wysiwyg-converter".